florist/tests/integration/api/test_train.py

```diff
@@ -13,9 +13,11 @@
 
 def test_train():
     # Define services
-    server_config = uvicorn.Config("florist.api.server:app", host="localhost", port=8000, log_level="debug")
+    server_config = uvicorn.Config(
+        "florist.api.server:app", host="localhost", port=8000, log_level="debug")
```

Comment:
nit: That's some ugly formatting. I find the format below better for readability:
```python
    server_config = uvicorn.Config(
        "florist.api.server:app", 
        host="localhost",
        port=8000,
        log_level="debug",
)
```